src/WP_CLI/LanguagePackUpgrader.php

```diff
@@ -46,9 +46,10 @@ public function generic_strings() {
 	 * @param string $package          The URI of the package. If this is the full path to an
 	 *                                 existing local file, it will be returned untouched.
 	 * @param bool   $check_signatures Whether to validate file signatures. Default false.
+	 * @param array  $hook_extra       Extra arguments to pass to the filter hooks. Default empty array.
 	 * @return string|\WP_Error The full path to the downloaded package file, or a WP_Error object.
 	 */
-	public function download_package( $package, $check_signatures = false ) {
+	public function download_package( $package, $check_signatures = false, $hook_extra = array() ) {
```

Comment:
Short array syntax is used for WP-CLI.
```suggestion
	public function download_package( $package, $check_signatures = false, $hook_extra = [] ) {
```